ethicml/evaluators/evaluate_models.py

```diff
@@ -471,8 +306,8 @@ def _gather_metrics(
         csv_file = _result_path(outdir, data_info.dataset_name, data_info.transform_name, topic)
         aggregator = ResultsAggregator(results_df)
         # put old results before new results -> prepend=True
-        aggregator.append_from_csv(csv_file, prepend=True)
+        # aggregator.append_from_csv(csv_file, prepend=True)
```

Comment:
Ah, this has to be restored.

```suggestion
        aggregator.append_from_csv(csv_file, prepend=True)
```